url.bs

```diff
@@ -346,49 +346,13 @@ obtain <var>host</var>'s <a for=host>registrable domain</a>, run these steps:
 </div>
 
-<p>Two <a for=/>hosts</a>, <var>A</var> and <var>B</var> are said to be
-<dfn for=host export>same site</dfn> with each other if either of the following statements are true:
-
-<ul class=brief>
- <li><p><var>A</var> <a for=host>equals</a> <var>B</var> and <var>A</var>'s
- <a for=host>registrable domain</a> is non-null.
-
- <li><p><var>A</var>'s <a for=host>registrable domain</a> is <var>B</var>'s
- <a for=host>registrable domain</a> and is non-null.
-</ul>
-
-<div class=example id=example-same-site>
- <p>Assuming that <code>suffix.example</code> is a <a for=host>public suffix</a> and that
- <code>example.com</code> is not:
-
- <ul>
-  <li><p><code>example.com</code>, <code>sub.example.com</code>, <code>other.example.com</code>,
-  <code>sub.sub.example.com</code>, and <code>sub.other.example.com</code> are all <a>same site</a>
-  with each other (and themselves), as their <a for=host>registrable domains</a> are
-  <code>example.com</code>.
-
-  <li><p><code>registrable.suffix.example</code>, <code>sub.registrable.suffix.example</code>,
-  <code>other.registrable.suffix.example</code>, <code>sub.sub.registrable.suffix.example</code>,
-  and <code>sub.other.registrable.suffix.example</code> are all <a>same site</a> with each other
-  (and themselves), as their <a for=host>registrable domains</a> are
-  <code>registrable.suffix.example</code>.
-
-  <li><p><code>example.com</code> and <code>registrable.suffix.example</code> are not
-  <a>same site</a> with each other, as their <a for=host>registrable domains</a> differ.
-
-  <li><p><code>suffix.example</code> is not <a>same site</a> with <code>suffix.example</code>, as
-  it is a <a for=host>public suffix</a>, and therefore has a null
-  <a for=host>registrable domain</a>.
- </ul>
-</div>
-
 <p class=warning>Specifications should prefer the <a for=/>origin</a> concept for security
-decisions. The notion of "<a for=host>public suffix</a>", "<a for=host>registrable domain</a>",
-and "<a>same site</a>" cannot be relied-upon to provide a hard security boundary, as the public
-suffix list will diverge from client to client. Specifications which ignore this advice are
-encouraged to carefully consider whether URLs' schemes ought to be incorporated into any decision
-made based upon whether or not two <a for=/>hosts</a> are <a>same site</a>. HTML's <a>same
-origin-domain</a> concept is a reasonable example of this consideration in practice.
+decisions. The notion of "<a for=host>public suffix</a>" and "<a for=host>registrable domain</a>"
+cannot be relied-upon to provide a hard security boundary, as the public suffix list will diverge
+from client to client. Specifications which ignore this advice are encouraged to carefully consider
+whether URLs' schemes ought to be incorporated into any decisions made, i.e. whether to use the
+<a>same site</a> or <a>schemelessly same site</a> concepts. HTML's <a>same origin-domain</a> concept
+is a reasonable example of this consideration in practice.
```

Comment:
Should we give this an ID and reference it from HTML?

Reply:
The note, you mean? I'm not sure what we would reference it for... personally I find the note pretty confusing, and was mostly trying to just preserve the meaning.

Reply:
The point is that we don't really want people to use (schemelessly) same site if they can avoid it in favor of origins. And I'd like HTML to also suggest that to some extent.

Reply:
Added an ID; will update HTML pull request now.

I was also thinking of removing the pointer to HTML's same-origin domain concept since I'm not sure I'd call it "reasonable". WDYT?

Reply:
Yeah, let's remove that.